test/Spec.hs

```diff
@@ -1,10 +1,58 @@
+import Data.List
+import Data.Function
 import Boids
 import Test.HUnit
-import Test.HUnit.Approx
 import System.Exit
 
+testIsWithinView :: Test
+testIsWithinView  = TestCase (assertBool "Seeing a boid behind me" (not $ isWithinView boid1 boid2))
```

Comment:
According to documentation of [assertBool] it's "The message that is displayed if the assertion fails"

[assertBool]: https://hackage.haskell.org/package/HUnit-1.5.0.0/docs/Test-HUnit-Base.html#v:assertBool